### PR TITLE
PlayerTracking HomePage, SignUpPage and MyData Page

### PR DESCRIPTION
**Pull request by TpSingh08**
The code is in react and all the three pages are linked through the working navbar, in Homepage the user can see all the information about the website, following that in the signup page one can fill the information and just after the user signup, the MyData page opens where the user can fill its personal details.

Note: first install the dependencies: npm install
then, npm run dev
Then all these pages can be easily accessed

### CODE REVIEW

File: Tarun/static/img/undraw-team-spirit-re-yl1v-1.svg
Comment (TpSingh08) [inline Tarun/static/img/undraw-team-spirit-re-yl1v-1.svg:1]: from undraw https://undraw.co/

File: Tarun/static/img/undraw-team-page-re-cffb-1.svg
Comment (TpSingh08) [inline Tarun/static/img/undraw-team-page-re-cffb-1.svg:1]: from undraw https://undraw.co/

File: Tarun/static/img/undraw-shared-goals-re-jvqd-1.svg
Comment (TpSingh08) [inline Tarun/static/img/undraw-shared-goals-re-jvqd-1.svg:1]: from undraw https://undraw.co/

File: Tarun/static/img/undraw-greek-freak-j-2-qb-1.svg
Comment (TpSingh08) [inline Tarun/static/img/undraw-greek-freak-j-2-qb-1.svg:1]: from undraw https://undraw.co/

File: Tarun/static/img/undraw-basketball-re-7701-1.svg
Comment (TpSingh08) [inline Tarun/static/img/undraw-basketball-re-7701-1.svg:1]: from undraw https://undraw.co/

File: Tarun/static/img/undraw-about-me-re-82bv-1.svg
Comment (TpSingh08) [inline Tarun/static/img/undraw-about-me-re-82bv-1.svg:1]: from undraw https://undraw.co/

File: Tarun/static/img/rectangle-20.png
Comment (TpSingh08) [inline Tarun/static/img/rectangle-20.png:1]: from https://www.townandcountrymag.com/leisure/sporting/a7165/badminton-olympics/

File: Tarun/static/img/image-2.png
Comment (TpSingh08) [inline Tarun/static/img/image-2.png:1]: from https://www.abc.net.au/news/science/2022-12-16/world-cup-2022-qatar-var-offside-technology/101762934

File: Tarun/static/img/rectangle-18.png
Comment (TpSingh08) [inline Tarun/static/img/rectangle-18.png:1]: from https://edition.cnn.com/2022/11/26/football/football-soccer-history-world-cup-spt-intl/index.html

File: Tarun/static/img/rectangle-19.png
Comment (TpSingh08) [inline Tarun/static/img/rectangle-19.png:1]: from https://www.britannica.com/sports/Cricket-World-Cup

File: Tarun/static/img/rectangle-1.png
Comment (TpSingh08) [inline Tarun/static/img/rectangle-1.png:1]: from https://stock.adobe.com/au/search?filters%5Bcontent_type%3Aphoto%5D=1&filters%5Bcontent_type%3Aillustration%5D=1&filters%5Bcontent_type%3Azip_vector%5D=1&filters%5Bcontent_type%3Avideo%5D=1&filters%5Bcontent_type%3Atemplate%5D=1&filters%5Bcontent_type%3A3d%5D=1&filters%5Bcontent_type%3Aaudio%5D=0&filters%5Binclude_stock_enterprise%5D=0&filters%5Bis_editorial%5D=0&filters%5Bfree_collection%5D=0&order=relevance&model_id=120193854&asset_id=258580472

File: Tarun/static/img/undraw-team-spirit-re-yl1v-1.svg
Comment (TpSingh08) [inline Tarun/static/img/undraw-team-spirit-re-yl1v-1.svg:1]: https://stock.adobe.com/au/search?filters%5Bcontent_type%3Aphoto%5D=1&filters%5Bcontent_type%3Aillustration%5D=1&filters%5Bcontent_type%3Azip_vector%5D=1&filters%5Bcontent_type%3Avideo%5D=1&filters%5Bcontent_type%3Atemplate%5D=1&filters%5Bcontent_type%3A3d%5D=1&filters%5Bcontent_type%3Aaudio%5D=0&filters%5Binclude_stock_enterprise%5D=0&filters%5Bis_editorial%5D=0&filters%5Bfree_collection%5D=0&order=relevance&model_id=120193854&asset_id=258580472

File: Tarun/static/img/ellipse-220.png
Comment (TpSingh08) [inline Tarun/static/img/ellipse-220.png:1]: from https://assets-global.website-files.com/5c7fa3041d33843df7f9354a/6564c455190ad32f16f48bbe_Progress-2.webp

File: Tarun/static/img/image-3.png
Comment (TpSingh08) [inline Tarun/static/img/image-3.png:1]: from https://www.playermaker.com/wp-content/uploads/2023/10/player.png

**Comment by ben-AI-cybersec** [comment]
Added citations as comments in GitHub instead of in code. Not integrated, created a new react app inside the existing react app. Closing PR

**Pull request closed without merging**